handlers_test.go

```diff
@@ -0,0 +1,152 @@
+package main
```

Comment:
I think we should remove this for now. I know it is boiler plate, but we should strive for a blackboox test, which will come when the refactoring starts.

Reply:
Yes. this handlers_test.go is just a placeholder for now for more white-box testing (unit tests).